### PR TITLE
Update some of our more 'dated' instructions

### LINKED ISSUE #11724: update petsc versions in documentation
## Rationale
Some instructions are getting dated. Time to bring them up-to-date.

## Description
Update PETSc version information on new documentation system.

## Impact
None

### PR DESCRIPTION
**Pull request by milljm**
Update the version of PETSc we were asking users to install.
Update the Windows 10 instructions. As well as alert the user as to its current supportability.

Closes #11724

### CODE REVIEW

**Comment by aeslaughter** [comment]
@milljm It would be simple to add an extension to use a file for the PETSc version, then in the markdown you could do something like:

```markdown
This is really cool, the current version of PETSc is !!package petsc!!, ...
```

I can help you get started, the "python/MooseDocs/extensions/acroynm.py" does something similar.

http://www.mooseframework.org/moose/utilities/MooseDocs/extensions/acronym.html#bef8d33b-4513-4f57-b8e8-ddeac08fcccd

**Comment by aeslaughter** [comment]
You can add this feature to the "package.py" extension.

**Comment by aeslaughter** [comment]
Another thing, while you are in here, the "getting started" instructions should be moved into modules/combined/doc/content... since they are specific for our website. That is, other applications (BISON, MASTODON, etc.) have their own set of instructions.

This will also allow us to remove the "packages" extension being available by default, since again this is really a MOOSE specific thing.

**Review by aeslaughter — CHANGES_REQUESTED** [review]
Please add comments to these classes and methods

File: framework/doc/content/getting_started/installation/cluster.md [hunk 1]
```diff
@@ -1,178 +1,205 @@
...
-  ```
+With the modulefile in place and the MODULEPATH variable set, see if our module is available for loading:
 
-The point being, some HPC clusters can have hundreds of available modules. While using this method,
-you can group and display certain modules pertinent to your MOOSE users at the end of stdout.
+```bash
+module avail
+```
 
+Verify that somewhere among all the other modules installed, you see: 'moose-dev-gcc' among them.
```
Comment (jwpeterson) [inline framework/doc/content/getting_started/installation/cluster.md]: typo: remove "among them" from end of sentence.

File: framework/doc/content/getting_started/installation/windows10.md [hunk 2]
```diff
@@ -1,42 +1,54 @@
...
+- Microsoft is now offering several different flavors of linux. Different versions means you need to choose wisely!
 
-After these two items are installed the following steps are required to complete the setup for the
-WSL to work for MOOSE. After these steps are completed then follow the install instructions for
-[Ubuntu](installation/ubuntu.md), but be sure to
-restart the bash session after installing the redistributable package.
+  - (at the time of this writing, we have installation packages for: Ubuntu and OpenSUSE Leap)
+!alert-end!
 
+Begin by performing the following external sets of instructions. Do note, that when it comes time to deciding what distribution of Linux you want, you should try and choose a flavor listed among our getting started pages for the smoothest installation process. This is not a requirement however, as MOOSE will run on just about any flavor of Linux if you are of the adventurous type!
```
Comment (jwpeterson) [inline framework/doc/content/getting_started/installation/windows10.md]: typo: "comes time to deciding" -> "comes time to decide"

**Review by jwpeterson — COMMENTED** [review]
@milljm I just found a few grammatical things I'd like to fix. If it's OK with you I will just push a commit directly to your branch that fixes them up.

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**